production/scrapers/ohio_population.R

```diff
@@ -51,55 +51,121 @@ ohio_population_pull <- function(x, exp_date = Sys.Date()){
 }
 
 ohio_population_restruct <- function(x){
```

Comment:
Can you rename function argument x to be more descriptive here? Maybe something like `raw_pdf`? It's easier for me to read when arguments/variables describe what type of information they hold.

Reply:
yup yup

Reply:
Writing this down here. Currently whenever we make a new scraper we base it off of this [outline](https://github.com/uclalawcovid19behindbars/covid19_behind_bars_scrapers/blob/master/prototyping/copy_scraper_outline.R). I wonder if we should just get in the habit of always replacing x's with more informative names.